e2e/LongHaul/service/HubEvents.cs

```diff
@@ -58,12 +59,20 @@ await foreach (PartitionEvent partitionEvent in eventHubClient.ReadEventsAsync(r
                                 {
                                     _logger.Trace($"HubEvents: known message source {eventMetadata.MessageSource}", TraceSeverity.Verbose);
                                 }
-                                break;
+                            }
+                            catch (Exception ex)
+                            {
+                                _logger.Trace(
+                                    $"Failed to convert event [{JsonConvert.SerializeObject(partitionEvent.Data.Properties)}] to DeviceEventProperties due to {ex}",
+                                    TraceSeverity.Warning);
+                            }
                         }
                     }
                     catch (Exception ex)
                     {
-                        _logger.Trace($"Failed to convert event [{JsonSerializer.Serialize(partitionEvent.Data.Properties)}] to DeviceEventProperties due to {ex}", TraceSeverity.Warning);
```

Comment:
The exception caught here is not necessarily caused by failed serial-/deserialization of `partitionEvent.Data.Properties`. It can be possibly caused by failures on `partitionEvent.Data.SystemProperties` as well. So fixed logging contents here.